standard-service/templates/postgres-user.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.postgres.user.enabled -}}
+apiVersion: postgres.coremaker.io/v1
+kind: PostgresUser
+metadata:
+  name: {{ template "coremaker-common-chart.fullname" . }}
+spec:
+  database:
+{{ toYaml .Values.postgres.user.databaseRef | indent 4 }}
```

Comment:
here should be database? or databaseRef? the example is with database? don't know which one is correct